src/main/java/com/dataloom/data/requests/GetEntitySetRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.dataloom.data.requests;
+
+import java.util.Set;
+import java.util.UUID;
+
+import com.dataloom.data.SerializationConstants;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Optional;
+
+public class GetEntitySetRequest {
+    @JsonProperty( SerializationConstants.SYNC_IDS )
+    private Optional<Set<UUID>> syncIds;
```

Comment:
This shouldn't be a client-side parameter. We will track state of syncIds (in progress/completed) on the backend and use that to generate list of syncIds we care about.